docs/getting_started/requirements.md

```diff
@@ -27,7 +27,7 @@ These setups are tested by QA and are generally recommended setups. For security
 
 |||
 |------|------|
-|Search|Solr (recommended; for performance, features and search quality):</br></br>Solr 6.x *Currently tested with Solr 6.6LTS*</br></br>Oracle Java/Open JDK: 8</br></br>*NOTE: If your Java version is higher than 8, you need to downgrade it to Oracle Java/Open JDK: 8.* |
+|Search|Solr (recommended; for performance, features and search quality):</br></br>Solr 7.x *Currently tested with Solr 7.7LTS*</br></br>Oracle Java/Open JDK: 8</br></br>*NOTE: If your Java version is higher than 8, you need to downgrade it to Oracle Java/Open JDK: 8.* |
```

Comment:
> *NOTE: If your Java version is higher than 8, you need to downgrade it to Oracle Java/Open JDK: 8.

This statement seems to be obsolete. Solr 7.7 documentation says that it could be run on Java 1.8+ (see https://lucene.apache.org/solr/guide/7_7/solr-system-requirements.html#java-requirements).